kafka-connect-hazelcast/src/main/scala/com/datamountaineer/streamreactor/connect/hazelcast/writers/HazelCastWriter.scala

```diff
@@ -78,8 +71,8 @@ class HazelCastWriter(client: HazelcastInstance, settings: HazelCastSinkSettings
       logger.debug("No records received.")
     } else {
       logger.debug(s"Received ${records.size} records.")
-      val converted = records.map(r => (r.topic, convert(r)))
-      converted.foreach({
+      val grouped = records.map(r => (r.topic, r))
```

Comment:
unless we run the code for each group in parallel I think looking for the writer on each record might be faster than grouping